.stylelintrc.json

```diff
@@ -18,5 +18,5 @@
     },
```

Comment:
### Your Readme :memo:
![Screenshot from 2022-09-20 16-03-55](https://user-images.githubusercontent.com/93039359/191243770-88388a2c-a9bc-48bd-b173-37a5d3c91de6.png)

- [ ] Kindly add a short description of your project in your readme file after the Project Title :+1:
- [ ] Kindly add technologies and languages in Built with section which you have used in this project :+1:

**This readme file is the documention of your project so please take it seriously because documenation shows how much you are following professionalism :thinking:**